lib/models/usuario.js

```diff
@@ -76,6 +80,64 @@ export default class Usuario extends Model {
     return omit(['contrasenia'], super.toJSON());
   }
 
+  async getTurnos() {
+    const turnos = await Turno.findAll({
+      where: {
+        usuarioId: this.id,
+      },
+      include: [
+        {
+          model: Actividad,
+          attributes: [
+            'id',
+            'nombre',
+            'fechaHoraInicio',
+            'fechaHoraFin',
+            'responsable',
+          ],
+          include: [
+            {
+              model: Espacio,
+              attributes: ['nombre'],
+              include: [
+                {
+                  model: Edificio,
+                  attributes: ['nombre'],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    });
+
+    return turnos;
+  }
+
+  async getIdsCarreras() {
+    const inscripciones = await this.getInscripcionCarreras({
+      attributes: ['idCarrera'],
+      where: { usuarioId: this.id },
+    });
```

Comment:
🔧 Micro-optimización: en un caso así donde no vas a usar ningún método del modelo, conviene ponerle `raw: true` a la query, para que directamente devuelva los valores sin instanciar las clases.

```suggestion
      raw: true
    });
```